scriptmodules/admin/joy2key.sh

```diff
@@ -15,11 +15,11 @@ rp_module_section="core"
 
 function _update_hook_joy2key() {
     # make sure joy2key is always updated when updating retropie-setup
-    rp_isInstalled "$md_id" && install_bin_joy2key
+    rp_isInstalled "$md_id" && depends_joy2key && install_bin_joy2key
```

Comment:
This could probably just be `rp_isInstalled "$md_id" && rp_callModule "$md_id"`

Reply:
`rp_isInstalled "joy2key" && rp_callModule "joy2key"` is fine too of course.

Reply:
yeah, it was an copied from the initial version where I've added the 'depends' on the `retropiemenu` update hook, but using the 1st version is shorter. Applied.